cluster-provision/k8s/scripts/node01.sh

```diff
@@ -6,7 +6,7 @@ source /tmp/scripts/cnis-map.sh
 
 version=`kubectl version --short --client | cut -d":" -f2 |sed  's/ //g' | cut -c2- `
 minor_version=`echo $version | cut -d"." -f2 `
-cni_manifest="/tmp/${CNI_MANIFESTS[$version]}"
+cni_manifest="/tmp/cni/${CNI_MANIFESTS[$version]}"
 
```

Comment:
In line 20 when applying  $cni_manifes '.yaml' is missing, it might fails

Reply:
not needed kubectl apply -f can accept directories and it will apply all yamls there.